spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-binder-sample/pom.xml

```diff
@@ -25,13 +39,66 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
+
+        <!-- Test-related dependencies. -->
+        <dependency>
+            <groupId>org.awaitility</groupId>
+            <artifactId>awaitility</artifactId>
+            <version>3.1.6</version>
```

Comment:
is there a way to put this 3.1.6 awaitility and 2.5 commons-io test dependencies in the BOM ? they're repeated like a billion times too

Reply:
Hmm, i'm not sure if adding it in our bom is appropriate since those deps are only needed in the tests of the samples. In practice users of spring-cloud-gcp may never use those libraries.

Reply:
yea that's a good point. I suppose it's just unfortunate then :P